### PR TITLE
Kirill.sitcevoy/library activity simulation

### PR DESCRIPTION
**Pull request by kievzenit**
Library activity simulation home task

### CODE REVIEW

File: Lab 1.2.1/Lab 1.2.1/LibraryUser.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                this.bookList[index] = value;
+            }
+        }
+
+        public LibraryUser()
+        {
+            this.FirstName = "No name";
+            this.LastName = "No last name";
+            this.Phone = "No phone";
+            this.Anonymous = true;
```
Comment (araka86) [inline Lab 1.2.1/Lab 1.2.1/LibraryUser.cs]: variable Anonymous-  where does he write? and 65 line

Comment (kievzenit) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs]: In task was said: implement two constructors with parameters and without. I think, only anonymous user can be without name, last name and phone (parameterless constructor). And what do you mean by he? Please, clear your question.

Comment (araka86) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs]: you don't have such a variable Anonymous

Comment (kievzenit) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs]: Oh, forgot to rename it((( fixed in commit ec77f715fd12673612e57b26bfd3abc546a93783

File: Lab 1.2.1/Lab 1.2.1/LibraryUser.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            this.LastName = lastName;
+            this.Phone = phone;
+            this.IsAnonymous = false;
+            this.BookLimit = bookLimit;
+            this.Id = LibraryUser.GetNextId();
+            this.bookList = new List<string>();
+        }
+
+        public void AddBook(string bookName)
+        {
```
Comment (SergiySimchuk) [inline Lab 1.2.1/Lab 1.2.1/LibraryUser.cs:74]: Maybe here should be check to over book limit?

**Review by araka86 — DISMISSED** [review]

**Review by SergiySimchuk — DISMISSED** [review]

File: Lab 1.2.1/Lab 1.2.1/LibraryUser.cs (hunk 2 above)
Comment (kievzenit) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs:74]: I have this check on lines 73-74

Comment (SergiySimchuk) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs:74]: Yes I saw. I wrote that comment today in the afternoon, it strange that it shown as 20minutes ago. Maybe I clicked wrong button ...  Anywhere, I approved this pull request)

Comment (kievzenit) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs:74]: Thanks for approving)))

File: Lab 1.2.1/Lab 1.2.1/LibraryUser.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        public string Phone { get; set; }
+        public int BookCount { get => this.bookList.Count; }
+
+        public readonly string FirstName;
+        public readonly string LastName;
+        public readonly int Id;
+        public readonly int BookLimit;
+        public readonly bool IsAnonymous;
+
+        private List<string> bookList;
```
Comment (Artya) [inline Lab 1.2.1/Lab 1.2.1/LibraryUser.cs]: this is hack, you have to use array.
this is need in order to see how usefull will be other .Net classes if compare with build-in types

**Review by Artya — CHANGES_REQUESTED** [review]

File: Lab 1.2.1/Lab 1.2.1/LibraryUser.cs (hunk 3 above)
Comment (kievzenit) [reply Lab 1.2.1/Lab 1.2.1/LibraryUser.cs]: Commit 8ed68811cec989030e8cd9b00e0e6001307a2b27

**Review by SergiySimchuk — APPROVED** [review]

**Review by Artya — APPROVED** [review]

**Pull request closed without merging**